lib/json/jsonfile.js

```diff
@@ -2,15 +2,21 @@
 
 const jsonFile = require('jsonfile')
 
+function write () {
+  jsonFile.writeFile.apply({ spaces: 2 }, arguments)
```

Comment:
It would definitely be simpler to do it that way; if no one's opposed, I'll change this.